options/rtdl/generic/linker.cpp

```diff
@@ -256,7 +256,7 @@ SharedObject *ObjectRepository::requestObjectAtPath(frg::string_view path, uint6
 
 	_parseDynamic(object);
 
-	_nameMap.insert(path, object);
+	_nameMap.insert(frg::string<MemoryAllocator>(path, getAllocator()), object);
```

Comment:
Construct a `string_view` based on `object->name` instead of a full copy of the string here (and other places that insert into `_nameMap`).